docs/README.md

```diff
@@ -116,4 +116,4 @@ You can [run the preview app locally](#running) or deploy it straight to a Herok
 An existing Heroku instance can be found at: [http://govuk-frontend-review.herokuapp.com/](http://govuk-frontend-review.herokuapp.com/)
 
 ## Publishing
-You need npm credentials to publish a new release. For the detailed publishing process see the [publishing  documentation](publishing.md).
+You need npm credentials to publish a new release. For the detailed publishing process see the [publishing  documentation](contributing/publishing.md).
```

Comment:
Whilst we're here, can we fix the double space between publishing and documentation?